mlem/contrib/heroku/meta.py

```diff
@@ -53,11 +62,12 @@ def ensured_app(self) -> HerokuAppMeta:
         return self.app
 
 
-class HerokuDeployment(MlemDeployment):
+class HerokuDeployment(MlemDeployment[HerokuState, HerokuEnv]):
```

Comment:
confused how to read this type hint. Why it could be `MlemDeployment[Y, Z]`? Could you please explain?

Reply:
This is for mypy and IDE help mostly. `MlemDeployment` is `Generic` and methods like `get_env` will return not arbitrary `MlemEnv`, but whatever you provide as second generic parameter (`HerokuEnv` in this case).